README.md

```diff
@@ -3,3 +3,8 @@
 [![Dependabot Status](https://api.dependabot.com/badges/status?host=github&repo=uw-it-edm/edm-zuul-proxy)](https://dependabot.com)
 
 develop: [![Build Status](https://travis-ci.org/uw-it-edm/edm-zuul-proxy.svg?branch=develop)](https://travis-ci.org/uw-it-edm/edm-zuul-proxy) [![Coverage Status](https://coveralls.io/repos/github/uw-it-edm/edm-zuul-proxy/badge.svg?branch=develop)](https://coveralls.io/github/uw-it-edm/edm-zuul-proxy?branch=develop)
+
+
```

Comment:
Can you add some direction about running on workstation, it seems you check for certain parameters in `isWorkstationEnvironment`